src/main/java/com/softserve/actent/repository/ChatRepository.java

```diff
@@ -4,8 +4,9 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.Optional;
+
 @Repository
 public interface ChatRepository extends JpaRepository<Chat, Long> {
-
-    Chat findOneById(Long id);
-}
+    Optional<Chat> findById(Long chatId);
```

Comment:
Signature of this method completely unnecessary here, as implementation of CrudRepository already has it.
https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/CrudRepository.html#findById-ID-